Vagrantfile

```diff
@@ -0,0 +1,52 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+
+  
+   config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+  # TODO: Install pyenv prerequisites
+  # TODO: Install pyenv
+   sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+  git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+
+  echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bash_profile
```

Comment:
Although this works, setting this in the `~/.profile` is better practice - the `.profile` file will be skipped since you created a `.bash_profile` and has some existing setup that could be usefu